CHANGELOG.md

```diff
@@ -1,26 +1,6 @@
-# Change Log
+# Changelog
 
-All notable changes to this project will be documented in this file.
-
-The format is based on
-[Keep a Changelog](http://keepachangelog.com)
-& makes a strong effort to adhere to
-[Semantic Versioning](http://semver.org).
-
-Tracking in this Changelog began for this project in version 3.25.0.
-If you're looking for changes from before this, refer to the project's
-git logs & PR history.
-
-The headers used in [Keep a Changelog](http://keepachangelog.com) are:
-
-- Added - for new features.
-- Changed - for changes in existing functionality.
-- Deprecated - for soon-to-be removed features.
-- Removed - for now removed features.
-- Fixed - for any bug fixes.
-- Security - in case of vulnerabilities.
-
-# [4.29.1](https://github.com/voxpupuli/beaker/compare/4.29.0...4.29.1) - 25-05-2021
+## [4.29.1](https://github.com/voxpupuli/beaker/tree/4.29.1) (2021-05-26)
```

Comment:
I would have expected to see an entry for 4.29.2.

Reply:
yes. I don't know yet why all merged PRs since the last release are ignored by the changelog generator :(